src/types/User.ts

```diff
@@ -68,6 +69,29 @@ export const UserMutations = extendType({
         return user;
       },
     });
+
+    t.field('Signup', {
+      type: 'User',
+      args: {
+        input: nonNull(
+          arg({
+            type: 'UserAuthInput',
+          }),
+        ),
+      },
+      resolve: async (source, { input }, context) => {
+        const hashedPassword = await bcrypt.hash(input.password, 12);
```

Comment:
jeg ville argumentere for at det var en god ide at lave en fil som "abtrahere" dette væk, sådan at man kan kalde auth.hash også er det bare skrevet at den altid hasher med 12 runder, i stedet for at sætte 12 på hvert kald